Output mangled

I made a mistake, and cibuildwheel was very helpful in erroring out (otherwise I would have missed the mistake), but the error message was not at all where I was expecting it:

```
2021-02-10T05:15:42.7037690Z ##[group]Run joerick/cibuildwheel@v1.9.0
2021-02-10T05:15:42.7038170Z with:
2021-02-10T05:15:42.7038500Z   package-dir: .
2021-02-10T05:15:42.7038900Z   output-dir: wheelhouse
2021-02-10T05:15:42.7039250Z env:
2021-02-10T05:15:42.7039620Z   SETUPTOOLS_SCM_PRETEND_VERSION: 
2021-02-10T05:15:42.7040240Z   CIBW_ENVIRONMENT: PIP_ONLY_BINARY=numpy SETUPTOOLS_SCM_PRETEND_VERSION=
2021-02-10T05:15:42.7040790Z   CIBW_TEST_EXTRAS: test
2021-02-10T05:15:42.7041250Z   CIBW_TEST_COMMAND: pytest {project}/tests
2021-02-10T05:15:42.7041760Z   CIBW_TEST_SKIP: pp*macos* *universal2:arm64
2021-02-10T05:15:42.7042170Z   CIBW_BUILD: 
2021-02-10T05:15:42.7042530Z   CIBW_SKIP: cp27-win* pp27-win*
2021-02-10T05:15:42.7042940Z   CIBW_MANYLINUX_I686_IMAGE: 
2021-02-10T05:15:42.7043340Z   CIBW_MANYLINUX_X86_64_IMAGE: 
2021-02-10T05:15:42.7043730Z   CIBW_ARCHS: universal2
2021-02-10T05:15:42.7044100Z ##[endgroup]
2021-02-10T05:16:02.6041070Z 
2021-02-10T05:16:02.6042070Z      _ _       _ _   _       _           _
2021-02-10T05:16:02.6042490Z  ___|_| |_ _ _|_| |_| |_ _ _| |_ ___ ___| |
2021-02-10T05:16:02.6044500Z [?25l[?25hcibuildwheel: No build identifiers selected: BuildSelector('' - 'cp27-win* pp27-win*')
2021-02-10T05:16:02.6045630Z |  _| | . | | | | | . | | | |   | -_| -_| |
2021-02-10T05:16:02.6046050Z |___|_|___|___|_|_|___|_____|_|_|___|___|_|
2021-02-10T05:16:02.6046300Z 
2021-02-10T05:16:02.6046710Z cibuildwheel version 1.9.0
2021-02-10T05:16:02.6046990Z 
2021-02-10T05:16:02.6047400Z Build options:
2021-02-10T05:16:02.6048170Z   platform: 'macos'
2021-02-10T05:16:02.6049220Z   architectures: {<Architecture.universal2: 'universal2'>}
2021-02-10T05:16:02.6050170Z   before_all: ''
2021-02-10T05:16:02.6050570Z   before_build: None
2021-02-10T05:16:02.6051000Z   before_test: None
2021-02-10T05:16:02.6051900Z   build_selector: BuildSelector('' - 'cp27-win* pp27-win*')
2021-02-10T05:16:02.6052710Z   build_verbosity: 0
2021-02-10T05:16:02.6054370Z   dependency_constraints: DependencyConstraintsPosixPath('/usr/local/Cellar/pipx/0.16.0.0/.cache/427fa552d5b62b6/lib/python3.9/site-packages/cibuildwheel/resources/constraints.txt'))
2021-02-10T05:16:02.6056250Z   environment: ParsedEnvironment(['PIP_ONLY_BINARY=numpy', 'SETUPTOOLS_SCM_PRETEND_VERSION='])
2021-02-10T05:16:02.6056980Z   manylinux_images: None
2021-02-10T05:16:02.6057870Z   output_dir: PosixPath('wheelhouse')
2021-02-10T05:16:02.6058740Z   package_dir: PosixPath('.')
2021-02-10T05:16:02.6060020Z   repair_command: 'delocate-listdeps {wheel} && delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}'
2021-02-10T05:16:02.6062130Z   test_command: 'pytest {project}/tests'
2021-02-10T05:16:02.6062950Z   test_extras: '[test]'
2021-02-10T05:16:02.6063400Z   test_requires: []
2021-02-10T05:16:02.6064340Z   test_selector: TestSelector('*' - 'pp*macos* *universal2:arm64')
2021-02-10T05:16:02.6064800Z 
2021-02-10T05:16:02.6065130Z Here we go!
2021-02-10T05:16:02.6065380Z 
2021-02-10T05:16:02.9515910Z ##[error]Process completed with exit code 1.
2021-02-10T05:16:02.9609080Z Cleaning up orphan processes
```

https://github.com/scikit-hep/boost-histogram/runs/1868826910?check_suite_focus=true

--- a/action.yml
+++ b/action.yml
@@ -19,10 +19,12 @@ runs:
 
     # This needs powershell-core because github.action path is a Windows-style
     # path on Windows.  Powershell-core understands both types of paths.
+    # Redirecting stderr to stdout to fix interleaving issue in Actions.
     - run: >
         pipx run
         --spec ${{ github.action_path }}
         cibuildwheel
         ${{ inputs.package-dir }}
         --output-dir ${{ inputs.output-dir }}
+        2>&1
       shell: pwsh

